README.md

```diff
@@ -47,9 +35,25 @@ The demo attempts to be feature complete according to documentation [here](https
 - TypingIndicator - to signify to the user that another user is typing in a given room.
 - UserHeader - to display useful information about a given user.
 
+## Running the Slack clone locally
```

Comment:
I'd say Usage is a pretty standard README title. What was the a reasoning behind this title change?

Reply:
Usage makes sense for a npm module, not so much a demo

Reply:
I like to reiterate the title sometimes